angular/core/components/thread_page/revision_history_modal/revision_history_modal.haml

```diff
@@ -1,18 +1,20 @@
-.revision-history-modal
-  .modal-header
-    %modal_header_cancel_button
-    %h1.lmo-h1{translate: "{{header}}"}
-  .modal-body.revision-history-modal__body
-    %h2.lmo-h2{ng-if: "commentRevision()"} {{model.authorName()}}
-    .lmo-wrap{ng-repeat: "version in model.versions() | orderBy: '-createdAt' track by version.id", id: "version-{{version.id}}"}
-      .revision-history-modal__revision
-        %h2.lmo-h2{ng-if: "discussionRevision()", translate: "{{threadTitle(version)}}"}
-        .revision-history-modal__thread-details
-          %span{ng-if: "commentRevision()", translate: "revision_history_modal.posted"}
-          %span{ng-if: "discussionRevision()", translate: "{{threadDetails(version)}}", translate-value-name: "{{version.authorOrEditorName()}}"}
-          %span> {{versionCreatedAt(version.createdAt)}}
-          %span{ng-if: "version.isCurrent(version)"}> (current)
-        .revision-history-modal__revision-body.lmo-markdown-wrapper{marked: "revisionBody(version)"}
+.lmo-blank
+  %loading{ng-if: "loadExecuting"}
```

Comment:
I wonder if we should wrap the modal body in a loading wrapper (rather than the whole thing), so folks can cancel out of the modal before it's done loading if they want to?